src/object-helpers.ts

```diff
@@ -4,11 +4,11 @@ const isObject = (value: any) =>
   Object.prototype.toString.call(value) === "[object Object]";
 
 function findPaginatedResourcePath(responseData: any): string[] {
-  const paginatedResourcePath = deepFindPathToProperty(
+  const paginatedResourcePath: string[] | null = deepFindPathToProperty(
     responseData,
     "pageInfo",
   );
-  if (paginatedResourcePath.length === 0) {
+  if (paginatedResourcePath === null || paginatedResourcePath.length === 0) {
```

Comment:
Did this line fix #58? The stacktrace and the description looks like it happens in `deepFindPathToProperty` implementation, not for the returned value.

>This causes currentValue.hasOwnProperty(searchProp) to throw TypeError: Cannot read properties of null (reading 'hasOwnProperty') error.

https://github.com/octokit/plugin-paginate-graphql.js/blob/17e58b6c3ec476e3877a9994b3b9f68a9ea6d8bf/src/object-helpers.ts#L26-L28

Reply:
https://github.com/octokit/plugin-paginate-graphql.js/pull/59/files#diff-be5e0092082abfc624a87b5ab130c1f22c2c68205637088e88268bbcc776e96dR26 looks focusing to there